WordPress/src/main/java/org/wordpress/android/util/CrashLogging.kt

```diff
@@ -1,172 +0,0 @@
-package org.wordpress.android.util
-
-import android.content.Context
-import android.preference.PreferenceManager
-import io.sentry.android.core.SentryAndroid
-import io.sentry.core.Sentry
-import io.sentry.core.SentryLevel
-import io.sentry.core.protocol.User
-import org.wordpress.android.BuildConfig
-import org.wordpress.android.R
-import org.wordpress.android.fluxc.store.AccountStore
-import org.wordpress.android.util.AppLog.T
-import org.wordpress.android.util.helpers.logfile.LogFileProvider
-import javax.inject.Inject
-import javax.inject.Singleton
-
-private const val EXTRA_UUID = "uuid"
-private const val EVENT_BUS_MODULE = "org.greenrobot.eventbus"
-private const val EVENT_BUS_EXCEPTION = "EventBusException"
-private const val EVENT_BUS_INVOKING_SUBSCRIBER_FAILED_ERROR = "Invoking subscriber failed"
-
-@Singleton
-class CrashLogging @Inject constructor(
-    private val accountStore: AccountStore,
-    private val encryptedLogging: EncryptedLogging
-) {
-    fun start(context: Context) {
-        SentryAndroid.init(context) { options ->
-            options.dsn = BuildConfig.SENTRY_DSN
-            options.cacheDirPath = context.cacheDir.absolutePath
-            options.isEnableSessionTracking = true // Release Health tracking
```

Comment:
also here: it's `true` by default: https://github.com/getsentry/sentry-java/blob/90984b2ba4f2f94c723156fa73a7bb1eac8ba1d5/sentry/src/main/java/io/sentry/SentryOptions.java#L199